src/main/java/com/ua/accommodation/model/Accommodation.java

```diff
@@ -1,4 +1,64 @@
 package com.ua.accommodation.model;
 
+import jakarta.persistence.Column;
+import jakarta.persistence.Entity;
+import jakarta.persistence.EnumType;
+import jakarta.persistence.Enumerated;
+import jakarta.persistence.GeneratedValue;
+import jakarta.persistence.GenerationType;
+import jakarta.persistence.Id;
+import jakarta.persistence.JoinColumn;
+import jakarta.persistence.JoinTable;
+import jakarta.persistence.ManyToMany;
+import jakarta.persistence.OneToOne;
+import jakarta.persistence.Table;
+import java.math.BigDecimal;
+import java.util.Set;
+import lombok.Data;
+import lombok.EqualsAndHashCode;
+import lombok.ToString;
+import org.hibernate.annotations.SoftDelete;
+
+@Entity
+@Table(name = "accommodations")
+@SoftDelete(columnName = "is_deleted")
+@Data
```

Comment:
Please change the getter and setter data
Optional: modify softDelete so that we have a consistent code style